src/multiqc/config.vsh.yaml

```diff
@@ -161,6 +161,12 @@ argument_groups:
         type: boolean_true
         description: |
           Disable coloured log output.
+      - name: "--cl_config"
+        type: file
+        required: false
+        description: |
+          YAML formatted string that allows to customize MultiQC behaviour like input file detection.
+        example: "qualimap_config: { general_stats_coverage: [20,40,200] }"
```

Comment:
I assume this is an example of the file's contents? Perhaps this could be moved the description instead?

Reply:
No, the argument should provide a YAML blob, not a file, but I now see that te type of the argument is wrong 🤷

Reply:
Aha :+1: